docs/app/Examples/elements/Label/Variations/LabelExampleCircular.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Label } from 'stardust'
+
+const LabelExampleCircular = () => {
+  const colors = [
```

Comment:
When we have data for examples, let's put it up top outside the scope of the example itself.